src/logging.rs

```diff
@@ -97,6 +105,50 @@ pub mod trace {
         }
     }
 
+    impl LevelHandle {
+        /// Returns a new `LevelHandle` without a corresponding filter.
```

Comment:
What is the corresponding filter that is omitted here?  To me the builder here looks the same as the one in `init_with_filter`

Reply:
The difference is that, rather than installing the subscriber as the default, it's dropped at the end of the function, so the handle doesn't point at anything.